### ISSUE TITLE
20170509_kiev_museum_great_patriotic_war the photo gallery type is incorrect

### ISSUE THREAD

**Issue by Illia2810** [scholokov/long-travel-2#4175]
Pre-conditions:
Open: https://test.long-travel.live/20170509_kiev_museum_great_patriotic_war/

Steps to reproduce:

    1. Open DevTools
    2. Looking for a div

Actual result:
the photo gallery type is incorrect Mosaic
Expected result:
the photo gallery type must be Justified Grid

Environment:
System: Windows 10 Pro
Browser: FireFox

Requirements:
https://github.com/scholokov/long-travel-2/wiki/4.-%D0%9F%D0%BE%D0%B4%D0%BE%D1%80%D0%BE%D0%B6 1.3.3.2.3

Attachment:
![зображення](https://github.com/scholokov/long-travel-2/assets/125801673/66e328f1-e238-4e0b-924f-5dd05fa74e88)

**Comment by scholokov** [comment]
according to requirements it is expected behaviour

**Comment by Illia2810** [comment]
no, on the contrary, according to the requirements, this trip is not the incorrect type of photo gallery
![зображення](https://github.com/scholokov/long-travel-2/assets/125801673/6e20c650-520c-4154-be9e-c5f126259e25)
![зображення](https://github.com/scholokov/long-travel-2/assets/125801673/5615a4a0-78e8-409a-9503-f838cec9b77a)

**Comment by scholokov** [comment]
ми переїхали на нові вимоги
https://qax.gitbook.io/qax-travel/requirements/requirements

**Comment by Illia2810** [comment]
Не є помилкою. В цій подорожі правильниий тип фото галереї
